public/app/features/alerting/unified/components/rule-editor/alert-rule-form/AlertRuleForm.tsx

```diff
@@ -85,6 +85,7 @@ export const AlertRuleForm = ({ existing, prefill }: Props) => {
     mode: 'onSubmit',
     defaultValues,
     shouldFocusError: true,
+    reValidateMode: 'onSubmit',
```

Comment:
I needed to add this prop to avoid revalidating any time a wrong fields was updated. Not sure if there is another way to solve this problem.